app-backend/common/src/main/scala/com/rasterfoundry/common/BacksplashConnectionFactory.scala

```diff
@@ -0,0 +1,10 @@
+package com.rasterfoundry.common
+
+import net.spy.memcached.{ClientMode, DefaultConnectionFactory}
+
+class BacksplashConnectionFactory extends DefaultConnectionFactory() {
+  override def getClientMode: ClientMode = Config.memcached.clientMode
+
+  override def getOperationTimeout: Long =
+    Config.memcached.timeout
```

Comment:
We previously used `Config.memcached.memcachedTimeoutMilliseconds`...did that even work?

Reply:
That did work, and still does, but this is pulling `Config` from `Common` -- in `backsplash-server` it has its own `Config` that uses a different variable (but pulls the same values). I'm going to see if I can clean up the different subprojects to be more consistent.